xt-prod-devel-rpi5-domd/recipes-kernel/linux/linux-raspberrypi_6.6.bbappend

```diff
@@ -3,3 +3,4 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 SRC_URI:append = " \
     file://optee.cfg \
 "
+SRC_URI:append = "${@bb.utils.contains('MACHINE_FEATURES', 'scmi', 'file://scmi-config.cfg', '', d)}"
```

Comment:
I think this separation is unnecessary for now we have scmi configuration split between meta-xt-rpi5 and meta-xt-prod-devel-rpi5. I think all configs needed to enable scmi should be in the same place

Reply:
done